lib/cc/analyzer/measurement_validations.rb

```diff
@@ -0,0 +1,16 @@
+module CC
+  module Analyzer
+    module MeasurementValidations
+      autoload :NameValidation, "cc/analyzer/measurement_validations/name_validation"
+      autoload :TypeValidation, "cc/analyzer/measurement_validations/type_validation"
+      autoload :ValueValidation, "cc/analyzer/measurement_validations/value_validation"
+      autoload :Validation, "cc/analyzer/measurement_validations/validation"
+
+      def self.validations
+        constants.sort.map(&method(:const_get)).select do |klass|
+          klass.is_a?(Class) && klass.superclass == Validation
+        end
```

Comment:
Do you think the literal array `[NameValidation, TypeValidation, ValueValidation]` might be preferable to iterating through `constants`? Besides being more straightforward, I wonder if there's a potential bug here if we added a helper module, for example.

Reply:
I like that this allows us to add future validations with less boilerplate code around which validations to run: we have the same bit of code in `IssueValidations`, so I'm also in favor of treating the two things the same.

>  I wonder if there's a potential bug here if we added a helper module

We have specs asserting the validations that are run, so that seems unlikely.